### PR TITLE
Add Extension: Mailsy

### PR DESCRIPTION
**Pull request by BalliAsghar**
## Description

With Mailsy users can quickly generate disposable email addresses, and view received Mails.

## Screencast

Screenshot 2022-06-03 at 5 11 11 pm

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by BalliAsghar** [comment]
https://user-images.githubusercontent.com/14165223/172030764-e210268f-9315-4c4e-b9ff-1fab4b2659a9.mov

### _Allowing Email to be Opened in Browser_

### Why

When an Email is opened in Raycast it only displays the text version of the email, but Sometimes Emails contain Html that is not viewable when opened in Raycast.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @BalliAsghar 👋 

Thanks for your contribution, I tried to test it but the first thing I encounter in all commands is this:

CleanShot 2022-06-09 at 14 40 18@2x

I'm not sure what to expect, could you look into this?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by BalliAsghar** [comment]
Hey @pernielsentikaer, 
I've committed some changes, this should fix the issues. Please have a look!

File: extensions/mailsy/src/mails.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          key={item.id}
+          title={item.subject}
+          subtitle={item.intro}
+          accessories={[
+            { text: `${new Date(item.createdAt).toLocaleString()}`, icon: Icon.Calendar },
+            { icon: Icon.Person, tooltip: item.from.name },
+          ]}
+          actions={
+            <ActionPanel>
+              <Action.Push title={"Open Email"} target={<Email id={item.id} />} />
```
Comment (pernielsentikaer) [inline extensions/mailsy/src/mails.tsx:52]:
```suggestion
              <Action.Push title={"Open Email"} icon={Icon.Envelope} target={<Email id={item.id} />} />
```
Missing icon

**Review by pernielsentikaer — APPROVED** [review]
Hi @BalliAsghar 👋 

Let's get it merged 🔥 

Thanks for your cooperation 💪

**Pull request merged**